module/TaskManagement/test/unit/TaskManagement/Controller/TransitionsControllerTest.php

```diff
@@ -77,7 +80,7 @@ public function testCreate(){
 		$this->controller->getTaskService()
 			->expects($this->once())
 			->method('findAcceptedTasksBefore')
-			->willReturn(array($this->task));
+			->willReturn(array(new ReadModelTask($this->task->getId())));
```

Comment:
Perchè il metodo 'findAcceptedTasksBefore' restituisce effettivamente un'array di oggetti di tipo TaskManagement\Entity\Task. Questa variazione è stata necessaria anche perchè la funzione 'taskClosedInfoMail', del service TaskManagement\Service\NotifyMailListener introdotto nel controller, riceve in ingresso un oggetto di tipo TaskManagement\Entity\Task. Comunque facendo la modifica relativa alla prima segnalazione questa variazione potrebbe essere ininfluente.